lib/java/desktop-environment/src/main/java/org/enso/desktopenvironment/Directories.java

```diff
@@ -2,9 +2,17 @@
 
 import java.io.IOException;
 import java.nio.file.Path;
+import org.enso.common.Platform;
 
 /** Provides information about user directories. */
 public sealed interface Directories permits LinuxDirectories, MacOsDirectories, WindowsDirectories {
+  static Directories getForCurrentPlatform() {
```

Comment:
Here we may also benefit from `CURRENT` being a compilation constant during NI build time. Make it a field:
```suggestion
  public static final Directories CURRENT = switch (...) {}
```

Reply:
Done in 48ae55b77fcfed3d289b299b4eb7e49590337f41